stories/LinterMessage.stories.tsx

```diff
@@ -13,160 +17,303 @@ const createMessage = (attributes = {}) => {
   });
 };
 
+const render = (children: JSX.Element) => {
+  return renderWithStoreAndRouter(children);
+};
+
+const renderWithMessageUid = (children: JSX.Element, messageUid: string) => {
+  return renderWithStoreAndRouter(children, undefined, [
```

Comment:
This call is confusing since you're using `undefined` to bypass optional arguments. I think it's time to change the signature to:

```tsx
renderWithStoreAndRouter(children, { location: createFakeLocation('...') });
```

The other calls could do:


```tsx
renderWithStoreAndRouter(children, { store: configureStore() });
```

Reply:
Why not just make them all object parameters? I.e., 
```tsx
renderWithStoreAndRouter({children, location: createFakeLocation('...') });
```

and 
```tsx
renderWithStoreAndRouter({children, store: configureStore() });
```

Or is that what you meant?

Reply:
@kumar303 You didn't reply, but looking at your suggestion I guess you're thinking that we always need to pass `children` and the other arguments are optional, so an object of optional arguments makes sense. I'll go ahead and make that change to this PR.

Reply:
Yes, I was more concerned about the optional arguments. Making them all object params would be fine, though.